playbooks/provisioning/openstack/sample-inventory/group_vars/OSEv3.yml

```diff
@@ -53,5 +53,7 @@ openshift_override_hostname_check: true
 ansible_become: true
 
 # # Flannel networking
+#osm_cluster_network_cidr: 10.128.0.0/14
 #openshift_use_openshift_sdn: false
 #openshift_use_flannel: true
```

Comment:
Currently, `openshift_use_flannel: true` must be present in `group_vars/all.yml` as well, otherwise the provisioning will fail.

The reason for this is that some of the prerequisite checks (and heat templates) are run on `localhost` as opposed to the openshift nodes. Which means they won't see `openshift_use_flannel` defined and the provisioning will fail.

I know it's not added in this patch, but it's something we should fix in a subsequent commit.

Reply:
hm, it has defaults for the heat templates

Reply:
What I mean is: the default is false. But when you only set `openshift_use_flannel: true` in `OSEv3.yml` and then set `openstack_private_data_network_name` in `all.yml` (which according to the sample inventory and documentation should be enough), you'll get a failure here:

https://github.com/openshift/openshift-ansible-contrib/blob/master/playbooks/provisioning/openstack/net_vars_check.yaml#L9-L14

The reason is because the net_vars_check run on localhost, which is not in the OSEv3 group, so it thinks you have `openshift_use_flannel` unset and will use the `False` default.